code/modules/projectiles/projectile/special/meteor.dm

```diff
@@ -1,3 +1,120 @@
+#define DEFAULT_METEOR_LIFETIME 1800
+#define MAP_EDGE_PAD 5
+
+
+GLOBAL_VAR_INIT(meteor_wave_delay, 625) //minimum wait between waves in tenths of seconds
+//set to at least 100 unless you want evarr ruining every round
+
+//Meteors probability of spawning during a given wave
+GLOBAL_LIST_INIT(meteors_normal, list(/obj/projectile/meteor/dust=3, /obj/projectile/meteor/medium=8, /obj/projectile/meteor/big=3, \
+						  /obj/projectile/meteor/flaming=1, /obj/projectile/meteor/irradiated=3)) //for normal meteor event
```

Comment:
Only indent up one layer.

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#tabs-not-spaces

Reply:
You also don't need those backslashes.